qabel_id/settings/base_settings.py

```diff
@@ -68,6 +71,29 @@
     )
 }
 
+# Login security
+
+AXES_COOLOFF_TIME = datetime.timedelta(minutes=1)
```

Comment:
The description of this PR says per 10 seconds. This LOC says per 1 minute. I like 1 minute but please update the description.